src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -309,6 +304,7 @@ export default {
 			this.password = ''
 			this.listable = CONVERSATION.LISTABLE.NONE
 			this.$store.dispatch('purgeNewGroupConversationStore')
+			this.$emit('close');
```

Comment:
all emits should be defined in the component:

You need to add a line on a root level (same as mixins, props, data e.t.c.)
```
emits: ['close'],
```

Maybe, lint:fix could do it automatically